students/Daniel_Spray/Lesson03/mailroom.py

```diff
@@ -0,0 +1,106 @@
+def main(b=(["William Gates, III","Mark Zuckerberg","Jeff Bezos","Paul Allen"],[653784.49,16396.10,877.33,708.42], [2,3,1,3], [], [])):
```

Comment:
your `main` should include a `while True` loop.

you should define your donors at the top of the file instead of the default value.. it also shouldn't be a tuple, as tuples are immutable data structures and you want to modify it.